src/App.js

```diff
@@ -1,20 +1,18 @@
-import './App.css';
 import React from 'react';
+import { Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import Navbar from './pages/Navbar';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = '';
-  }
-
-  render() {
-    return (
-      <div className="App">
-        <Calculator />
-      </div>
-    );
-  }
-}
-
+const App = () => (
+  <>
+    <Navbar />
+    <Routes>
+      <Route exact path="/home" element={<Home />} />
```

Comment:
- [ ] It would be nice if the home path be just `"/"` instead of the "/home" you are having. The reason to have just empty "/" is because, by default, that will route to http://localhost:3000/ which will serve as the first page that loads when the app runs and that is the route that is to be your homepage.
- [ ] Currently, the localhost route "/" is showing a blank page when the app loads and it's because there is no page set for that route in your App.js routes. Please remove the "home" in both the App.js and the Navbar.js so you have your homepage set to the default "/". By so doing, you will now have 3 pages and not 4 pages with one being empty 👍🏽